src/lib/viewers/media/DashViewer.js

```diff
@@ -182,6 +186,78 @@ class DashViewer extends VideoBaseViewer {
         this.player.load(this.mediaUrl, this.startTimeInSeconds).catch(this.shakaErrorHandler);
     }
 
+    /**
+     * Handles the buffering events from shaka player
+     *
+     * @see {@link https://shaka-player-demo.appspot.com/docs/api/shaka.Player.html#.event:BufferingEvent}
+     * @param {Object} object - BufferingEvent object
+     * @param {boolean} object.buffering - Indicates whether the player is buffering or not
+     */
+    bufferingHandler({ buffering }) {
+        const tag = Timer.createTag(this.options.file.id, MEDIA_METRIC.totalBufferLag);
```

Comment:
Should we pull out `totalBufferLag` value into a variable, since it's used so much?